CONTRIBUTING.md

```diff
@@ -27,9 +27,7 @@ branch should correspond to an associated issue opened on the main repository
 git clone https://github.com/quipucords/quipucords
 cd quipucords
 git checkout -b issues/my_issue_#
-pip install -r dev-requirements.txt
-docker-compose up -d
-docker-compose exec qpc-db psql -c 'alter role qpc with CREATEDB'
+poetry install
```

Comment:
should this be --with dev since the pip was with dev-requirements.txt ?

Reply:
I thought so, but then I realised `poetry install` also installs dev dependencies.

Reply:
Gotcha, It's just that you had `poetry install --with dev` in the .github/workflows/test.yml file so I thought it was necessary.

Reply:
yeah... lemme remove that to avoid confusion